src/editor/Collaboration.js

```diff
@@ -12,14 +12,23 @@ define(function (require, exports, module) {
             // immediately ask for camera access
             autoRequestMedia: false
         });
+        //To be moved to the bramble API.
+        var hash = window.location.hash.replace(/^#/, "");
+        var m = /&?collaboration=([^&]*)/.exec(hash);
```

Comment:
This doesn't seem to be working. 
I think it has to do with the fact that our url is 
`http://localhost:8000/src/hosted.html?collaboration=asdds`
and not 
`http://localhost:8000/src/hosted.html/?collaboration=asdds`

Opening 
`http://localhost:8000/src/hosted.html/?collaboration=asdds`
doesn't seem to open hosted.html.

Another similar approach I found is :
`URLSearchParams`

```js
var params = window.location.href.split("?").pop();
        this.room = new URLSearchParams(params).get("collaboration") || Math.random().toString(36).substring(7);
        
```